hal/halx86/generic/x86bios.c

```diff
@@ -149,9 +149,9 @@ HalInitializeBios(
 NTSTATUS
 NTAPI
 x86BiosAllocateBuffer(
-    _In_ ULONG *Size,
-    _In_ USHORT *Segment,
-    _In_ USHORT *Offset)
+    _Inout_ ULONG *Size,
+    _Out_ USHORT *Segment,
+    _Out_ USHORT *Offset)
```

Comment:
Please also update the prototype in ndk/halfuncs.h with the new annotations.

Reply:
Done